apps/admin/backend/src/util/workspace.ts

```diff
@@ -22,16 +16,11 @@ export interface Workspace {
 export function createWorkspace(root: string): Workspace {
   const resolvedRoot = resolve(root);
   const dbPath = join(resolvedRoot, 'data.db');
-  const uploadsPath = join(resolvedRoot, 'uploads');
```

Comment:
Is this just old, unused code?

Reply:
AFAICT. @eventualbuddha what was the `uploads` subdirectory in VxAdmin backend supposed to be used for?

Reply:
Probably where we temporarily stored things uploaded via express+multer. Old code.